build.sbt

```diff
@@ -0,0 +1,56 @@
+import scala.sys.process._
+import play.sbt.PlayImport.PlayKeys._
+
+val libName         = "play-frontend-pta"
+val silencerVersion = "1.7.2"
+val npmBuild        = taskKey[Unit]("npm-build")
+val npmTest         = taskKey[Unit]("npm-test")
+
+lazy val root = Project(libName, file("."))
+  .enablePlugins(PlayScala, SbtTwirl)
+  .disablePlugins(PlayLayoutPlugin)
+  .settings(
+    name := libName,
+    majorVersion := 0,
+    scalaVersion := "2.12.13",
+    libraryDependencies ++= LibDependencies(),
+    TwirlKeys.templateImports := templateImports,
+    npmTest := {
+      val exitCode = ("npm install" #&& "npm test").!
+      if (exitCode != 0) {
+        throw new MessageOnlyException("npm install and test failed")
+      }
+    },
+    npmBuild := {
+      val exitCode = ("npm install" #&& "npm run build").!
```

Comment:
@matthewmascord might be worth changing this and the one above to use [npm ci](https://docs.npmjs.com/cli/v7/commands/npm-ci) rather than npm install

Reply:
Done